base/cmake_package.py

```diff
@@ -34,7 +35,7 @@ def configure(ctx, stage_args):
     else:
         conf_lines.append('-DCMAKE_BUILD_TYPE:STRING=Release')
 
-    if ctx.parameters['platform'] == 'Darwin':
+    if stage_args.get('empty_osx_deployment_target', False):
```

Comment:
@certik - I think this change also requires a change in `cmake_package.yaml` because setting `empty_osx_deployment_target: true` has currently no effect. I think what is missing is this:

``` diff
diff --git a/base/cmake_package.yaml b/base/cmake_package.yaml
index 3cd08de..7464694 100644
--- a/base/cmake_package.yaml
+++ b/base/cmake_package.yaml
@@ -15,6 +15,7 @@ build_stages:
   - name: configure
     after: setup_builddir
     debug: {{debug}}
+    empty_osx_deployment_target: {{empty_osx_deployment_target}}
 
   - name: make
     after: configure
```